homeassistant/helpers/trigger_template_entity.py

```diff
@@ -176,18 +176,42 @@ def restore_attributes(self, last_state: State) -> None:
                 extra_state_attributes[attr] = last_state.attributes[attr]
             self._rendered[CONF_ATTRIBUTES] = extra_state_attributes
 
+    def _render_availability_template(self, variables: dict[str, Any]) -> None:
```

Comment:
Should we change this to return a boolean to skip the rest of rendering in case it's not available?